EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/GeneralMessageHandler/GeneralMessageHandler.cs

```diff
@@ -43,14 +45,31 @@ public async Task ProcessCommand(string userInput)
                 await messageService.SendMessage("[Sarcasm]: I dont remember...\n" +
                 $"[Statement]:That was a joke human.\n[Recalling]: I remember {remember}");
                 break;
-
+            case "changerpg":
+                await ChangeRpgs(commandParams[1]);
```

Comment:
I think we want commandParams not just the first one or is the plan to only have 1 word 
"sprawl", "sprawlcustom", "bladesinthedark", "vtm3.5", "vtmv20"

Reply:
tbh I was thinking either camelCase or kebab-case